docs/reference/taxii-feeds.md

```diff
@@ -1,7 +1,32 @@
 # Taxii feeds
+In OpenCTI, you can import TAXII collections from the Data tab, then Ingestion under the TAXII Feeds tab.
 
-!!! tip "Under construction"
+![TAXII Feeds panel.png](assets/TAXII_Feeds_panel.png)
 
-    We are doing our best to complete this page. 
-    If you want to participate, don't hesitate to join the [Filigran Community on Slack](https://community.filigran.io) 
-    or submit your pull request on the [Github doc repository](https://github.com/OpenCTI-Platform/docs).
+## TAXII feeds management interface
+
+On the first page, you'll find the TAXII Feeds already set up. For each one, you'll see the name, the url of the source TAXII server, the version, the running status and the current feed status.
+
+Please note that OpenCTI only supports TAXII version 2.1.
```

Comment:
You can add that if they want to ingest TAXII version 2.0, they can use the connector TAXII